app/user.go

```diff
@@ -48,7 +48,8 @@ const (
 )
 
 func (a *App) CreateUserWithToken(user *model.User, token *model.Token) (*model.User, *model.AppError) {
-	if err := a.IsUserSignUpAllowed(); err != nil {
+	err := a.IsUserSignUpAllowed()
```

Comment:
Any reason to break this up? The lines below already use `nErr`.

Reply:
Don't remember but I fixed it now :)